0xaaiden/explorer-1
# Batches explore

Summary
Added recent batches tab to the Homepage table

Screenshot:
image

To Test
Open homepage, click on Recent Batches Tab.
Check it display recent batch orders
Click on Tx. Hash should redirect to order page
Hovering over tokens should take to etherscan token page
Embedded Menu should have 3 items (Etherscan, Cow Explorer, and copy Tx Id)
